edit-post/components/plugin-more-menu-item/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/element';
+import { Slot, Fill, withContext, MenuItemsGroup, MenuItemsItem } from '@wordpress/components';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Name of slot in which the sidebar should fill.
+ *
+ * @type {String}
```

Comment:
Lower-case `string`. Apparently the ESLint rule doesn't work very well for `@type`:

https://github.com/WordPress/gutenberg/blob/db7fcb91fad933c094ae43da4fc10568c4747f78/eslint/config.js#L153